Disaster-Response-Platform/backend/Services/feedback_service.py

```diff
@@ -0,0 +1,61 @@
+from Models.feedback_model import Feedback
+from Database.mongo import MongoDB
+from bson.objectid import ObjectId
+
+from Services.build_API_returns import *
+
+
+# Get the needs collection using the MongoDB class
+feedback_collection = MongoDB.get_collection('feedback')
```

Comment:
Instead of using collection names as a string, you can use the enumerated strings in the entity_type model. (Means that the enumeration in the model will also use the collection names.